docker-compose.yaml

```
@@ -5,4 +5,4 @@ services:
     image: paidyinc/one-frame
     container_name: one-frame-service
     ports:
-      - "8080:8080"
\ No newline at end of file
+      - "8090:8080"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)